pkg/v1/tkg/utils/common_test.go

```diff
@@ -413,6 +413,61 @@ var _ = Describe("CompareVMwareVersionStrings", func() {
 			Expect(err).To(MatchError("invalid version string"))
 		})
 	})
+
+	Context("When buildVersion includes more information about fips etc", func() {
```

Comment:
nit: should you need to update to address the linter errors, instead of being so literal (fips...) in the Context.. line
maybe something along the lines of "when buildVersion includes -suffix ..".